client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -52,15 +65,22 @@ const CustomsStep = ( props ) => {
 CustomsStep.propTypes = {
 	siteId: PropTypes.number.isRequired,
 	orderId: PropTypes.number.isRequired,
+	packageDescriptions: PropTypes.objectOf( PropTypes.string ).isRequired,
+	expanded: PropTypes.bool.isRequired,
```

Comment:
`expanded` can equal undefined when the form is first opened, which causes a React warning:
> Failed prop type: The prop `expanded` is marked as required in `CustomsStep`, but its value is `undefined`

Either this prop needs to change to not required, or we need to add a default value of `false` in the initial state. Would prefer to have an initial value of `false` to make it explicit.

Reply:
I changed it to not required, to be consistent with the rest of the steps.